roles/wildfly_systemd/tasks/main.yml

```diff
@@ -203,6 +203,9 @@
         - daemon_reload is defined or daemon_reload_env is defined
         - daemon_reload.changed or daemon_reload_env.changed
 
+    - name: "Ensure service is started"
+      ansible.builtin.include_tasks: selinux_config.yml
+
```

Comment:
@RanabirChakraborty I don't think you have set the pid_file location. It should match where the wildfly_systemd place the PID file (and be overrideable).